<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/images/register.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tb_page">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>471</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_login">
     <attribute name="title">
      <string>登录</string>
     </attribute>
     <widget class="QLabel" name="lb_tel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>手机号</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_password">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_tel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>12345678900</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_password">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>123x</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_clear">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_submit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ta_register">
     <attribute name="title">
      <string>注册</string>
     </attribute>
     <widget class="QLabel" name="lb_tel_register">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>80</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>手机号</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_password_register">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_tel_register">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_passwor_register">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_submit_register">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_clear_register">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_name_register">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_name_register">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>昵称</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
